src/content/ecosystem-explorer/projects/destor.md

```diff
@@ -0,0 +1,33 @@
+---
+title: DeStor
+created-on: 2024-07-30T16:28:00.000Z
+updated-on: 2024-07-30T16:28:00.000Z
+published-on: 2024-07-30T16:28:00.000Z
+image:
+  src: /assets/images/destor-logo-white.svg
+category: storage
+subcategories:
+  - storage-services
+description: DeStor is a leader in decentralized storage solutions, leveraging
+  blockchain technology to provide secure, scalable, and verifiable storage
+  services.
+website: https://destor.com/
+tech:
+  - filecoin
+  - ipfs
+twitter: https://x.com/FilecoinDeStor
+video-url: <iframe allowtransparency="true" title="Wistia video player"
+  allowFullscreen frameborder="0" scrolling="no" class="wistia_embed"
+  name="wistia_embed" src="https://fast.wistia.net/embed/iframe/vq3xqwmsoj"
+  width="400" height="225"></iframe>
```

Comment:
@matthendrian - This can only be an URL so `https://fast.wistia.net/embed/iframe/vq3xqwmsoj`